app/src/services/course.services.ts

```diff
@@ -72,13 +89,25 @@ const createSection = async (data: any, id: any, token: string) => {
   );
 }
 
+
+const deleteCourse = async (id: any, token: string) => {
+  console.log("Deleted course with id: " + id)
+  return await axios.delete(
+      `${BACKEND_URL}/api/courses/delete/${id}`,
```

Comment:
Do not have actions in the endpoint.
This would be api/courses/coverImage/:id

A thing to note, that I am not sure on, should cover-image be pluralized?